project/Build.scala

```diff
@@ -222,7 +222,9 @@ object Courier extends Build with OverridablePublishSettings {
     s";++$currentScalaVersion;project scala-generator;$publishCommand" +
     s";++$sbtScalaVersion;project sbt-plugin;$publishCommand" +
     s";++$sbtScalaVersion;project scala-runtime;$publishCommand" +
-    s";++$currentScalaVersion;project scala-runtime;$publishCommand"
+    s";++$currentScalaVersion;project scala-runtime;$publishCommand" +
+    s";++$sbtScalaVersion;project scala-fixture;$publishCommand" +
```

Comment:
I was pattern matching agains `scala-runtime` here. If you're confident this isn't needed, I'm happy to remove.